### PR TITLE
[stable24] Make it possible to insert heif/heic images

### PR DESCRIPTION
**Pull request by eneiluj**
Manual backport of https://github.com/nextcloud/text/pull/2972

I don't know why but explicit heic/heif mime types have to be added in the `<input>`.
They are not shown with `image/*`.

If such files are inserted and the preview provider is not enabled (or does not work), Text behaves just like if we insert a corrupted image.

### CODE REVIEW

File: src/components/MenuBar.vue [hunk 1]
```diff
@@ -24,7 +24,7 @@
 	<div class="menubar" :class="{ 'show': isVisible, 'autohide': autohide }">
 		<input ref="imageFileInput"
 			type="file"
-			accept="image/*"
+			accept="image/*, image/heic, image/heif"
```
Comment (vinicius73) [inline src/components/MenuBar.vue:27]: Why it is necessary?

Comment (eneiluj) [reply src/components/MenuBar.vue:27]: > I don't know why but explicit heic/heif mime types have to be added in the `<input>`.
They are not shown with image/*.

**Review by juliushaertl — APPROVED** [review]

**Pull request merged**